main.py

```diff
@@ -21,62 +21,64 @@
 motor_front_left.set_stop_action('brake')
 hub.motion_sensor.reset_yaw_angle()
 _select_trip = 0
+MAX_SPEED = 100
 
 ##############################################################
 # Tunable Constants
-GYRO_TURN_FAST_SPEED = 25
-GYRO_TURN_SLOW_SPEED = 10
+GYRO_TURN_FAST_SPEED = 20
+GYRO_TURN_SLOW_SPEED = 8
 BLACK_MIDDLE = 40
 BLACK_EDGE = 50
+SLOW_DOWN_ANGLE_BUFFER = 30
 ##############################################################
 
 ##############################################################
 # Available Functions
-# straight(degrees_to_move = 500, speed = 20)
-# two_wheel_move(left_degrees = 652, right_degrees = 398, speed = 20)
-# acquire_line(speed = 20)
+# straight(degrees_to_move=500, speed=20)
+# two_wheel_move(left_degrees=652, right_degrees=398, speed=20)
+# acquire_line(speed=20)
 # rot_motion()
-# line_follower(move_degrees = 1000, speed = 20)
-# gyro_turn(input_angle = 90, relative = False, timeout = 6, left_or_right = OneWheelTurn.BOTH)
-# grind(left_speed = 20, right_speed = 20, run_seconds=3)
+# line_follower(move_degrees=1000, speed=20)
+# gyro_turn(input_angle=90, relative=False, timeout=6, left_or_right=OneWheelTurn.BOTH)
+# grind(left_speed=20, right_speed=20, run_seconds=3)
 #vrooom()
 ###############################################################
 
-class OneWheelTurn:
+class TurnType:
     BOTH = 0
     LEFT = 1
     RIGHT = 2
 
 def motor_front_move():
```

Comment:
@EricGamer1127 @FortressBen what is "motor_front_move" used for? This is a bad name, it doesnt tell anything about why the motor is moving.  One can already tell its moving the front motors from the source, but why?   

I expected to see it used in one of the trips-- but i dont see where its used.

I think we should talk to the team about how we define funcitons that move the front motors for trips. There are a couple of options:

(1) define them inside the trip, so its already clear that they support that trip
(2) use names like trip_with_the_crates_grab_the_crates()